### DISCUSSION TITLE
[FRIGATE] Potential driver issue with Intel N100 systems using hardware acceleration

### DISCUSSION THREAD

**Discussion by bobloadmire** [tteck/Proxmox#2816] [General]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

I have been having a lot of issues trying to get HW acceleration working correctly on Proxmox on my N100 system. The Frigate dev has been helping me try and toubleshoot the issue, and he mentioned this: https://github.com/blakeblackshear/frigate/discussions/11009#discussioncomment-9157798

"I notice the script installs va-driver-all which is just intel-media-driver meanwhile the actual frigate container installs intel-media-driver-non-free so that might be part of the issue. Could be other things missing too"

I see you guys are actively developing this script, so I thought I would bring this to attention in case it helps. Thanks for the script!

### What settings are you currently utilizing?

Advanced Settings

### Which Linux distribution are you employing?

Debian 11

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

Link to full discussion: https://github.com/blakeblackshear/frigate/discussions/11009#discussion-6520970

### Please provide detailed steps to reproduce the issue.

Enable VINNO and VAAPI on Frigate on Proxmox on N100 system using this script.

**Comment by willgryan** [comment]
I am also experiencing this issue. Intel N100 on Proxmox 8.2.2 / Kernel 6.8.4.3-pve. HW acceleration for ffmpeg works fine. 

Updating to last Intel compute-driver in the LXC container per here seemed to resolve it: https://github.com/blakeblackshear/frigate/discussions/10785#discussioncomment-9258468
